src/canvas/canvasUtils.js

```diff
@@ -112,94 +112,100 @@ export const trimFlamegraphText = (
 
 export function getHoveredEvent(
   schedulerCanvasHeight: number,
-  data: $ReadOnly<ReactProfilerData>,
+  data: $ReadOnly<ReactProfilerDataV2>,
   flamechart: $ReadOnly<FlamechartData>,
   state: PanAndZoomState,
 ): ReactHoverContextInfo | null {
   const {canvasMouseX, canvasMouseY, offsetY} = state;
 
-  if (canvasMouseX < LABEL_FIXED_WIDTH || canvasMouseY < HEADER_HEIGHT_FIXED) {
+  if (canvasMouseY < HEADER_HEIGHT_FIXED) {
     return null;
   }
 
-  if (canvasMouseY + offsetY < schedulerCanvasHeight) {
-    const adjustedCanvasMouseY = canvasMouseY - HEADER_HEIGHT_FIXED + offsetY;
-    let priorityMinY = HEADER_HEIGHT_FIXED;
-    let priorityIndex = null;
-    let priority: ReactPriority = 'unscheduled';
-    for (let index = 0; index < REACT_PRIORITIES.length; index++) {
-      priority = REACT_PRIORITIES[index];
+  if (canvasMouseY + offsetY < HEADER_HEIGHT_FIXED + EVENT_ROW_HEIGHT_FIXED) {
```

Comment:
Comments on these switch branches would make it a lot quicker to grok what's going on

Reply:
I actually put the comment in the next line, to be consistent with where the comment had to go in the `else if` block below. Maybe it's not clear enough; let me think of another way

Reply:
Okay, I've tried addressing this in https://github.com/MLH-Fellowship/scheduling-profiler-prototype/pull/40 by breaking this big function. The broken function should also better parallels the broken `renderCanvas` so I think it's much clearer